pyproject.toml

```diff
@@ -0,0 +1,25 @@
+[tool.poetry]
+name = "termpdf"
+version = "0.1.0"
+description = "Graphical pdf reader that works inside the kitty terminal"
+authors = ["David Sanson"]
+
+[tool.poetry.dependencies]
+python = "^3.10"
```

Comment:
I think this is far too restrictive - it's still working for me when I change this to `^3.8`, which is still an officially supported Python version